core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1220,6 +1220,10 @@ private[spark] class DAGScheduler(
         _.getProperty(SparkContext.SPARK_JOB_GROUP_ID) == groupId
       }
     }
+    if (activeInGroup.isEmpty && !cancelFutureJobs) {
+      logWarning(s"Cancel job group $groupId failed due to " +
```

Comment:
`Failed to cancel job group $groupId. Cannot find active jobs for it.`

Reply:
Done.
Thanks!